### PR TITLE
[Biography] Added show view

### PR DESCRIPTION
**Pull request by thomasbisignani**

### CODE REVIEW

File: src/Admin/ExecutiveOfficeMemberAdmin.php [hunk 1]
```diff
@@ -167,10 +165,8 @@ protected function configureListFields(ListMapper $listMapper)
                 'label' => 'Dernière mise à jour',
             ])
             ->add('_action', null, [
-                'actions' => [
-                    'edit' => [],
-                    'delete' => [],
-                ],
+                'virtual_field' => true,
```
Comment (ottaviano) [inline src/Admin/ExecutiveOfficeMemberAdmin.php]: you can keep `edit` and `delete` actions and add you custom action (link) with:
```
->add('_action', null, [
  'actions' => [
    'link' => [
      'template' => 'template_path.html.twig', // with only one link
    ],
    'edit' => [],
    'delete' => [],
  ],
])
```

File: templates/page/le-mouvement/notre-organisation.html.twig [hunk 2]
```diff
@@ -38,50 +38,62 @@
...
-        </div>
-    {% endif %}
-
-    {% if (executiveOfficeMembers) %}
-        <div id="executive-office-members">
-            <h2 class="text--center">Le Bureau Exécutif</h2>
-            <ul class="team">
-                {% for member in executiveOfficeMembers %}
+    <div id="biography">
+        {% if (executiveOfficer) %}
```
Comment (ottaviano) [inline templates/page/le-mouvement/notre-organisation.html.twig]: you can remove the `(` & `)` for `if` on twig

File: templates/social_share/social_networks.html.twig [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% if object.facebookProfile %}
+    <a class="link--no-decor" href="{{ object.facebookProfile }}" target="_blank"><i class="fa fa-facebook"></i></a>
```
Comment (ottaviano) [inline templates/social_share/social_networks.html.twig]: what about adding `@AssertUrl` to these properties?

File: front/style/content/_movement.scss [hunk 4]
```diff
@@ -166,14 +166,30 @@
     width: 100%;
 }
 
-#executive-officer .team {
-    justify-content: center !important;
+#biography {
+    .executive-officer {
+        .team {
+            justify-content: center !important;
```
Comment (JeremieSamson) [inline front/style/content/_movement.scss]: Why do you use !important ?

Comment (thomasbisignani) [reply front/style/content/_movement.scss]: It's an override of this property : https://github.com/EnMarche/en-marche.fr/blob/master/front/style/content/_movement.scss#L51

**Pull request merged**